client/extensions/woocommerce/app/order/order-customer/index.js

```diff
@@ -70,21 +89,23 @@ class OrderCustomerInfo extends Component {
 				closeDialog={ this.toggleDialog( false ) }
 				isBilling
 				isVisible={ 'billing' === this.state.showDialog }
+				siteId={ siteId }
 				updateAddress={ this.updateAddress( 'billing' ) }
 			/>,
 			<CustomerAddressDialog
 				key="dialog-shipping"
 				address={ shipping }
 				closeDialog={ this.toggleDialog( false ) }
 				isVisible={ 'shipping' === this.state.showDialog }
+				siteId={ siteId }
 				updateAddress={ this.updateAddress( 'shipping' ) }
 			/>,
 		];
 	};
 
 	render() {
-		const { isEditing, order, translate } = this.props;
-		if ( ! order ) {
+		const { countries, isEditing, loadedLocations, order, translate } = this.props;
+		if ( ! order || ! loadedLocations ) {
```

Comment:
NM about my prior question about showing a placeholder - I'm seeing a pattern that was already established here of just returning null.